util.c

```diff
@@ -5322,3 +5286,159 @@ int util_is_wlan(char *device)
   return slist_getentry(config.net.wlan.devices, device) ? 1 : 0;
 }
 
+
+/*
+ * Write log message.
+ *
+ * level is a bitmask determining the destination to log to.
+ *
+ * Add a time stamp when the destination has the LOG_TIMESTAMP flag set.
+ */
+void util_log(unsigned level, char *format, ...)
```

Comment:
well, one idea. I see that very often ( >90% ) format contains `\n` at the end. I also find it quite common to have endline at the end of log message. Do you consider adding it by default there? Only case I see from this review is something like `checking....` and then `done` which can be done on next line with something like `checking finished`